zerver/tests/test_markdown.py

```diff
@@ -1400,21 +1400,25 @@ def test_topic_links_ordering_by_priority(self) -> None:
                     realm=realm,
                     pattern="http",
                     url_template="http://example.com/",
+                    order=1,
                 ),
                 RealmFilter(
                     realm=realm,
                     pattern="b#(?P<id>[a-z]+)",
                     url_template="http://example.com/b/{id}",
+                    order=2,
                 ),
                 RealmFilter(
                     realm=realm,
                     pattern="a#(?P<aid>[a-z]+) b#(?P<bid>[a-z]+)",
                     url_template="http://example.com/a/{aid}/b/{bid}",
+                    order=3,
                 ),
                 RealmFilter(
                     realm=realm,
                     pattern="a#(?P<id>[a-z]+)",
                     url_template="http://example.com/a/{id}",
+                    order=4,
                 ),
```

Comment:
Without setting `order` explicitly, these linkifiers are not consistently ordered the order they are in this list.